bigtable/tests/unit/test_cluster.py

```diff
@@ -18,6 +18,17 @@
 import mock
 
 
+def _make_credentials():
+    import google.auth.credentials
+
+    class _CredentialsWithScopes(
+            google.auth.credentials.Credentials,
+            google.auth.credentials.Scoped):
+        pass
+
+    return mock.Mock(spec=_CredentialsWithScopes)
```

Comment:
Rather than copying `_make_credentials`, we should be importing it from a shared location (e.g., `from ._utils import _make_credentials`